src/TahoeLAFS/Storage/Backend/Memory.hs

```diff
@@ -181,6 +182,9 @@ instance Backend MemoryBackend where
         return result
     readImmutableShares _ _ _ _ _ = error "readImmutableShares got bad input"
 
+    renewLease :: MemoryBackend -> StorageIndex -> [LeaseSecret] -> IO ()
+    renewLease _ _ _ = pure mempty -- XXX not even
```

Comment:
Maybe worth filing a ticket mentioning that renewLease isn't implemented in either backend.  I don't think we care now but perhaps someday we will.